_episodes/15-transferring-files.md

```diff
@@ -8,183 +8,107 @@ objectives:
+or from the cluster. There are several options for transferring data between computing 
+resources, from command line options to GUI programs, which we will cover here.
 
-## Grabbing files from the internet
+## Download files from the internet using wget
 
-To download files from the internet, the easiest tool to use is `wget`. The syntax is relatively
-straightforwards: `wget https://some/link/to/a/file.tar.gz` We've actually done this before to
-download our example files:
+One of the most straightforward ways to download files is to use `wget`. Any file 
+that can be downloaded in your web browser with an accessible link can be downloaded 
+using `wget`. This is a quick way to download datasets or source code. 
+
+The syntax is: `wget https://some/link/to/a/file.tar.gz`. For example, download the 
+lesson sample files using the following command:
 
 ```
 [remote]$ wget https://hpc-carpentry.github.io/hpc-intro/files/bash-lesson.tar.gz
 ```
 {: .bash}
 
-## Transferring single files and folders with scp
 
-To copy a single file to or from the cluster, we can use `scp`. The syntax can be a little complex
-for new users, but we'll break it down here:
+##  Compressing files
 
-To transfer *to* another computer:
-```
-[local]$ scp /path/to/local/file.txt yourUsername@remote.computer.address:/path/on/remote/computer
-```
-{: .bash}
+Let's look at the file we just downloaded:
 
-To download *from* another computer:
 ```
-[local]$ scp yourUsername@remote.computer.address:/path/on/remote/computer/file.txt /path/to/local/
+[remote]$ ls
+bash-lesson.tar.gz
 ```
-{: .bash}
-
-Note that we can simplify doing this by shortening our paths. On the remote computer, everything
-after the `:` is relative to our home directory. We can simply just add a `:` and leave it at that
-if we don't care where the file goes.
-
-```
-[local]$ scp local-file.txt yourUsername@remote.computer.address:
-```
-{: .bash}
 
-To recursively copy a directory, we just add the `-r` (recursive) flag:
-
-```
-[local]$ scp -r some-local-folder/ yourUsername@remote.computer.address:target-directory/
-```
-{: .bash}
-
-## Transferring files interactively with sftp
+You may recognize that this is a compressed file, or an *archive*, by the file extension `tar.gz`. 
+Sometimes it is convenient to compress files to make file transfers easier. 
+The larger the file, the longer it will take to transfer. Moreover, we can compress a whole bunch of little files
+into one big file to make it easier on us (no one likes transferring 70000 little files)!
 
-`scp` is useful, but what if we don't know the exact location of what we want to transfer? Or
-perhaps we're simply not sure which files we want to transfer yet. `sftp` is an interactive way of
-downloading and uploading files. Let's connect to a cluster, using `sftp`- you'll notice it works
-the same way as SSH:
+Let's extract this file with the `tar` command:
 
 ```
-sftp yourUsername@remote.computer.address
-```
-{: .bash}
```

Comment:
Please restore the `{: .bash}` tag for syntax highlighting,
and separate out the output into its own block with a `{: .output}` tag.